src/cart/crt_init.c

```diff
@@ -68,6 +68,7 @@ dump_envariables(void)
 		"D_LOG_FILE", "D_LOG_FILE_APPEND_PID", "D_LOG_MASK", "DD_MASK",
 		"DD_STDERR", "DD_SUBSYS", "CRT_TIMEOUT", "CRT_ATTACH_INFO_PATH",
 		"OFI_PORT", "OFI_INTERFACE", "OFI_DOMAIN", "CRT_CREDIT_EP_CTX",
+		"CRT_POST_INIT", "CRT_POST_INCR",
```

Comment:
Probably better to change to D_ type envariables as we were trying to get away from CRT_ prefixes

Reply:
changed